docs/design/faireness-effectiveness-statistic.md

```diff
@@ -0,0 +1,57 @@
+# Volcano Statistic And Metrics
+
+@jamesjiang; April 24, 2020
+
+## Motivation
+
+Currently, Volcano does not have statistics for cluster admin to evaluate the job wait fairness and every user and job receive the same treatment.
+
+Volcano also dose not have statistics for platform administrator to evaluate resources usage when job wait latency is so high. 
+
+Cluster admin can get the result when he finish tuning plugins of cluster to fit his own 
+
+
+## Design
+
+1. Fairness is very important for cluster admin to keep a balance for preempt and resource reserve.
+2. Effectiveness is also an important factor for cluster admin to consider for resource reserve.
+
+### Fairness
+
+> When scheduling is in high latency and retry rates and e2e_scheduling_latency_standard_deviation is large 
+
+1. job_running_duration_coefficient_of_variation: Job Running Coefficient Of Variation
+2. e2e_scheduling_latency_coefficient_of_variation: Job Lagency Coefficient Of Variation
+3. job_retry_rates: job_retry_count / job_running_count
+
+
+#### Machine Learning (Keep Fairness)
+
+> how to set plugins and parameters to make the following value small is machine learning work 
+
+- job_running_duration_coefficient_of_variation (1D) <Low is better>
+- e2e_scheduling_latency_coefficient_of_variation (1D) <Low is better>
+- job_retry_rates (1D) <Low is better>
+
+### Effectiveness
+
+> When scheduling latency is bad, tell cluster admin to add resource or tuning schedule plugin or parameters, if the scheduling is efficient, then add resources, if scheduling is not efficient then tuning plugin and parameters through other metrics.
+> The effectiveness is to make a balance to fairness, because sometimes fairness will cause less effectiveness, and some scene does not care about effectiveness.
+
+1. total_resouces_average_usage: CPU, Memory, GPU (3 metrics)
+2. node_usage_coefficient_of_variation: CPU, Memory, GPU (3 metrics)
```

Comment:
per node usage? Feel metric server may record the same metrics, but we can do it in volcano to facilitate usage